tests/features/support/steps.js

```diff
@@ -27,3 +32,13 @@ defineStep(/^I (?:enter|fill|input|supply|type).*? "(.*?)" in.*? (`.*`)$/, (valu
 
 defineStep(/^I (?:activate|click).*? (`.*`)$/, selectorChain =>
   client.click(nestedSelector(selectorChain)));
+
+defineStep(/^I should be on.*? `(.*)`$/, pageName => {
+  let expectedUrl;
+  if (pageName.startsWith('/')) {
+    expectedUrl = `${url}${pageName}`;
+  } else {
+    expectedUrl = pages[pageName];
+  }
+  client.assert.urlEquals(expectedUrl);
```

Comment:
Use `expect` as elsewhere instead of `assert`.

Reply:
I couldn't find the expect equivalent https://github.com/nightwatchjs/nightwatch/issues/861, I'm not sure expect suppports urlEquals

Reply:
The last comment there shows an alternative, i.e.
```js
  client.url(currentUrl => {
    client.expect(currentUrl.value).to.eql(expectedUrl);
  });
```